apps/dashboard/config/spring.rb

```diff
@@ -0,0 +1,6 @@
+%w[
+  .ruby-version
+  .rbenv-vars
+  tmp/restart.txt
+  tmp/caching-dev.txt
+].each { |path| Spring.watch(path) }
```

Comment:
While Spring is helpful for speeding up running tests in development, we ran into problems with it in the past in our environment. Are we reintroducing Spring? Or does this not have any impact on whether Spring is used or not?